colorbleed/plugins/publish/submit_publish_job.py

```diff
@@ -159,3 +159,7 @@ def process(self, instance):
         response = requests.post(url, json=payload)
         if not response.ok:
             raise Exception(response.text)
+
+        # Temporary key name, deadlineSubmissionJob was already taken
+        if instance.data("runSlapComp", False):
```

Comment:
I don't think this is doing what you intended. Probably meant: `instance.data.get("runSlapComp", False)`?